### PR TITLE
added secure_referer_check to riak_kv schema

### PR DESCRIPTION
**Pull request by joedevivo**
A community user reported this setting missing from the schema.

### CODE REVIEW

**Comment by frli** [comment]
:thumbsup:

File: priv/riak_kv.schema [hunk 1]
```diff
@@ -285,3 +285,24 @@
   {commented, "1d"}, %% no default, it's undefined.
   {level, advanced}
 ]}.
+
+%% @doc In Riak 1.2 we added some measures to counteract cross-site scripting
```
Comment (bookshelfdave) [inline priv/riak_kv.schema]: maybe "Measures were added to Riak 1.2 to counteract cross-site scripting..."

Comment (joedevivo) [inline priv/riak_kv.schema]: sure!

**Comment by bookshelfdave** [comment]
+1, eunits pass, with the exception on:

```
module 'riak_kv_buckets_fsm'
module 'riak_kv_buckets_fsm_sup'
module 'riak_kv_cinfo'
module 'riak_kv_console'
module 'riak_kv_coverage_filter'
riak_kv_crdt:441: eqc_test_ (module 'riak_kv_crdt')...*failed*
::undef
```

**Pull request merged**